modules/assim.batch/R/gelman_diag.R

```diff
@@ -0,0 +1,83 @@
+#' @title Calculate Gelman diagnostic on moving window
+#'
+#' @author Alexey Shiklomanov
+#' @param x MCMC samples, of class \code{mcmc} or \code{mcmc.list}
+#' @param width_fraction Fractional width of moving window. Default=0.1.
+#' @param width Width of moving window. Default is niter(x)*width_fraction
+#' @param njump Number of windows to calculate over
```

Comment:
Actually ended up dropping that comment altogether, now that there are multiple methods.